crates/types/Cargo.toml

```diff
@@ -10,9 +10,15 @@ description = "Polymorphic Types for an Experimental Async SDR Runtime for Heter
 keywords = ["sdr", "radio", "runtime", "async", "acceleration"]
 categories = ["asynchronous", "concurrency", "hardware-support", "science", "wasm"]
 
+[features]
+default = []
+# Provides support for converting seify types into Pmt variants.
+seify = ["dep:seify"]
+
 [dependencies]
 dyn-clone = "1.0"
 num-complex = { version = "0.4", features = ["serde"] }
+seify = { version = "0.16", default-features = false, optional = true }
```

Comment:
Is there a better way of keeping this seify version in sync with the other crates? I don't see the use of `xyz.workspace = true` in any other `Cargo.toml` files, so assumed there's a reason for that.

Reply:
Tbh, I didn't know that this is possible. But it seems that workspace dependencies cannot be optional. So it might not be applicable, but I'm not sure.

Reply:
FWIW, you can tweak the "inheritance" with specific overrides like this:

```toml
seify = { workspace = true, optional = true }
```

I personally find TOML very weird...